### ISSUE TITLE
High Memory usage Mac App

### ISSUE THREAD

**Issue by vordenken** [MhdHejazi/CoronaTracker#22]
The mac app uses a lot of ram. Starting at about ~110 MB all the way to ~300 MB.
Is this due to it being a catalyst app? It doesnt need that much ram on the iphone..

**Comment by MhdHejazi** [comment]
> Is this due to it being a catalyst app?

I suspect that in the beginning. I started by inspecting the app memory but found nothing interesting except the big blocks of memory allocated and not released.

After paying more attention to the used memory while using the app it was clear the memory is allocated by the map view. So, I removed it to make sure it's the culprit, and sure enough, the memory went down to under 50mb.

I thought it's a bug with Mac Catalyst, so I created a new project and only added a map view to the empty screen, and after running the app and playing with the map I quickly reached 300mb.

Finally, I wanted to see if it's only a problem with Catalyst apps, so I opened Apple Maps and just zoomed in/out to different locations, and in 15 seconds I crossed the 300mb line.

So,  it seems to be an issue with macOS that affects all apps using the MapKit. Not sure if we can do anything here.
